tool/tctl/common/collection.go

```diff
@@ -1689,17 +1692,99 @@ func (p *pluginResourceWrapper) UnmarshalJSON(data []byte) error {
 			p.PluginV1.Spec.Settings = &types.PluginSpecV1_Datadog{}
 		case settingsEmailAccessPlugin:
 			p.PluginV1.Spec.Settings = &types.PluginSpecV1_Email{}
+		case settingsAWSIdentityCenter:
```

Comment:
There are bunch of validation we do for create operation, curious where do we handle such input validation for edit operation? For example, the region fields, SCIM URL, requiring both SCIM endpoint and credential is updated at once etc.

Reply:
See gravitational/teleport.e#6105